demo/README.md

```diff
@@ -0,0 +1,7 @@
+# Uast viewer example
```

Comment:
For docs, we use `doc`, `tutorials` and `examples` folders, so in this case, the `demo` folder should be renamed to `examples`.

Reply:
@dpordomingo it's not example**s**. It's an example. One. Do you think it will be better to rename demo -> example?

Reply:
agree! We can improve it later and make proper examples for end users.

For now, I use this app for development and it provides a good example of API during code review.